Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -321,7 +321,7 @@ private extension CoreDataManager {
         var persistentStoreRemovalError: Error?
         do {
             try persistentContainer.persistentStoreCoordinator.destroyPersistentStore(at: storeURL, type: .sqlite, options: nil)
-            NotificationCenter.default.post(name: .StorageManagerDidResetStorage, object: self)
+            NotificationCenter.default.post(name: .StorageManagerDidInvalidateCachedData, object: self)
```

Comment:
nit: what's the part that invalidates cached data? maybe it could more specific to what this function does like `StorageManagerDidDropDatabase`/`StorageManagerDidDropStorage`?

Reply:
Done in e7ce7524303acb1a26ab9e441b06349908da4f7a.